src/Dockerfile

```
@@ -12,6 +12,14 @@ RUN wget -nv https://download.oracle.com/otn_software/linux/instantclient/19600/
 RUN unzip instantclient-basiclite-linux.x64-19.6.0.0.0dbru.zip
 RUN cp -r  instantclient_19_6/* /usr/local/lib
 
+# Install sqlite3 from source, the default version for this Ubuntu-based image is too old
+RUN wget -c https://www.sqlite.org/2021/sqlite-autoconf-3360000.tar.gz
+RUN tar xzf sqlite-autoconf-3360000.tar.gz
+WORKDIR /tmp/sqlite-autoconf-3360000
+RUN ./configure
+RUN make
+RUN make install
```

```suggestion
RUN ./configure --disable-static --disable-static-shell && make install distclean
```

This should make it compile much faster and make the final image a few tens of MBs smaller.

---

Nice improvement, I'll add it!